Does not compile on current version of `mpv`

```
videowidget.cpp:57:77: error: too many initializers for ‘mpv_opengl_init_params’
   57 |     mpv_opengl_init_params gl_init_params{get_proc_address, nullptr, nullptr};
      |                                                                             ^
```

Found related issues:
https://github.com/Zren/mpvz/issues/12
https://github.com/flaviotordini/minitube/issues/217

--- a/src/video/videowidget.cpp
+++ b/src/video/videowidget.cpp
@@ -54,7 +54,7 @@ static void* get_proc_address(void* ctx, const char* name) {
 }
 
 void VideoWidget::initializeGL() {
-    mpv_opengl_init_params gl_init_params{get_proc_address, nullptr, nullptr};
+    mpv_opengl_init_params gl_init_params{get_proc_address, QOpenGLContext::currentContext()};
     mpv_render_param display{MPV_RENDER_PARAM_INVALID, nullptr};
 #ifdef X11_SUPPORT
     if(QX11Info::isPlatformX11()) {

